### PR TITLE
chore: ESLint improvements

### PR DESCRIPTION
**Pull request by Mitcheljager**
This PR adds more ESLint configurations to make our code more consistent. I'm not going to go through each change. The vast majority of changes you will see are for spaces in interpolated values `${ thing } -> ${thing}`. I wanted to do the same for svelte blocks `{ #each } -> {#each}` but I couldn't find the configuration. Many other rules are rules we already kind of had in place, but now they will be enforced.

### CODE REVIEW

**Comment by netux** [comment]
> I wanted to do the same for svelte blocks `{ #each } -> {#each}` but I couldn't find the configuration.

[mustache-spacing](https://sveltejs.github.io/eslint-plugin-svelte/rules/mustache-spacing/)?

**Comment by Mitcheljager** [comment]
@netux That's it!

File: app/javascript/src/get-snippet.js [hunk 1]
```diff
@@ -35,6 +35,6 @@ function loadSnippet(event, element) {
     })
     .catch(error => {
       console.error(error)
-      ideElement.innerHTML = `Failed to load Snippet <br>${error}`
+      ideElement.innerText = `Failed to load Snippet <br />${error}`
```
Comment (netux) [inline app/javascript/src/get-snippet.js:38]: Wouldn't this cause the `<br />` to be set as part of the text?

Comment (Mitcheljager) [reply app/javascript/src/get-snippet.js:38]: Ah yup, of course

**Pull request merged**